Gemfile

```diff
@@ -5,12 +5,3 @@ gem 'rack'
 gem 'grape'
 gem 'cucumber'
 gem 'aruba'
```

Comment:
I would actually remove the entire `Gemfile` since with the changes you made in the `Makefile` we don't need it anymore.